basics/3_executor_options/app.py

```diff
@@ -6,19 +6,19 @@
 from jina import Flow
 from docarray import Document, DocumentArray
 from helper import docs, print_search_results
-
+       
 flow = (
     Flow()
     .add(
         uses="jinahub://SpacyTextEncoder",
         # Change to "medium" model for better encoding
-        uses_with={"model_name": "en_core_web_md"},
+        uses_with={"model_name": "en_core_web_md", 'traversal_paths': 'r'},
         name="encoder",
         install_requirements=True
     )
     .add(
         # Switch `uses` to pull from Docker, so we can pass `metas`
-        uses="jinahub+docker://SimpleIndexer",
+        uses="jinahub://SimpleIndexer",
```

Comment:
```suggestion
        uses="jinahub://SimpleIndexer/v0.x",
```

Change version to whatever version works. Otherwise this may break more easily in future if Executor is updated